railties/lib/rails/command/spellchecker.rb

```diff
@@ -8,49 +8,9 @@ def suggest(word, from:)
           if defined?(DidYouMean::SpellChecker)
             DidYouMean::SpellChecker.new(dictionary: from.map(&:to_s)).correct(word).first
           else
-            from.sort_by { |w| levenshtein_distance(word, w) }.first
+            ""
```

Comment:
It should return `nil` if `DidYouMean::SpellChecker` isn't defined.

https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/railties/lib/rails/commands/server/server_command.rb#L265-L271

https://github.com/rails/rails/blob/2af458a10d336ec15a5734dba27709c2f8eb745b/railties/lib/rails/generators.rb#L275-L282

Reply:
Ah yes, I should have looked there. 🙇

Reply:
@kamipo It's returning nil now. Thanks!